interface/themes/style_light.css

```diff
@@ -80,12 +80,6 @@ a:hover {
   font-weight: bold;
 }
 
-.alert {
-  font-size: 70%;
-  color: #FF0000;
-  text-decoration: none;
-}
```

Comment:
All styles should already have the alert class, I just wanted this one to look better.